src/OpenToolkit.Mathematics/Matrix/Matrix4.cs

```diff
@@ -1379,6 +1380,7 @@ public static Matrix4 Mult(Matrix4 left, Matrix4 right)
         /// <param name="left">The left operand of the multiplication.</param>
         /// <param name="right">The right operand of the multiplication.</param>
         /// <param name="result">A new instance that is the result of the multiplication.</param>
+        [MethodImpl((MethodImplOptions)512 | MethodImplOptions.NoInlining)]
```

Comment:
What does the 512 do here?

Reply:
Uh, it's AggressiveOptimization.

But uh, I don't think I intended for this to be in this PR so I should probably revert this change.